autoload/memolist.vim

```diff
@@ -162,6 +162,10 @@ function! memolist#list()
   endif
 endfunction
 
+function! memolist#files()
+  return systemlist('find ' . s:escarg(g:memolist_path) . ' -type f')
```

Comment:
Is ` -type f` can availbale in windows?
I think `glob()` is more portable.

Reply:
I think it is indeed better to use glob in Vim script, so I will modify it and create a new PR.